USERS.md

```diff
@@ -156,6 +156,7 @@ Currently, the following organizations are **officially** using Argo Workflows:
 1. [Workiva](https://www.workiva.com/)
 1. [Voyager](https://investvoyager.com/)
 1. [Zhihu](https://www.zhihu.com/)
+1. [Iflytek](https://www.iflytek.com/)
```

Comment:
@berlinsaint can you move the alphabet order?

Reply:
> @berlinsaint can you move the alphabet order?

ok